prow/cmd/deck/static/prow/prow.ts

```diff
@@ -604,6 +604,7 @@ function redraw(fz: FuzzySearch, pushState: boolean = true): void {
         }
         displayedJob++;
         const r = document.createElement("tr");
+        // State column
```

Comment:
Was able to create a function for one of the columns to simplify some of the code, however, there are other pieces that are too complex to do so. Comments were left still, because I do still believe they provide good notice to a developer.